lib/simplifyExpression/basicsSearch/removeAdditionOfZero.js

```diff
@@ -1,30 +1,32 @@
-const clone = require('../../util/clone');
+const {canApplyRule, applyRule} = require('math-rules');
```

Comment:
Do you think we should keep the `require` syntax or move to babel's import syntax on this new branch?